openrtb2/loss_reason_code.go

```diff
@@ -1,41 +0,0 @@
-package openrtb2
-
-// 5.25 Loss Reason Codes
-//
-// Options for an exchange to inform a bidder as to the reason why they did not win an impression.
-type LossReasonCode int64
```

Comment:
Weird, looks like this one was not used anywhere.

Reply:
Ahh!!!

This enum is used for `${AUCTION_LOSS}` macro.

https://github.com/InteractiveAdvertisingBureau/openrtb/blob/master/OpenRTB%20v3.0%20FINAL.md#list--loss-reason-codes-

TODO:

- update LossReason in openrtb3 (some reasons were added)
- document this in release notes

Reply:
https://github.com/InteractiveAdvertisingBureau/openrtb/blob/master/OpenRTB%20v3.0%20FINAL.md#appendixb_changelog

Spec v3.0 is supposed to be adopted by the industry. It has a damn CHANGELOG. Version is not bumped, but changes like this one are introduced:

> 3.0 | September 2021 | Clarified loss reason: Loss Reason Code 206 was renamed  from Bundle to Store ID to keep the intent of this loss reason  consistent across OpenRTB 2.x and 3.x.

It basically refers to https://github.com/InteractiveAdvertisingBureau/openrtb/commit/5a6aac5491fc097e546c36bfb34461182542d24e where loss reasons were simply "shifted".

Versioning?.. Semver?.. Nah, why bother!

Reply:
I'll update openrtb3 loss reasons, but the more I get back to openrtb the more disappointing it is.

</venting>

Reply:
Submitted https://github.com/InteractiveAdvertisingBureau/openrtb/issues/102 - don't think it'll change soon, but at least it's reported.